_code-samples/non-fungible-token/js/list-nft-pages-and-buy-offers.js

```diff
@@ -47,6 +50,6 @@
     }
     client.disconnect()
     // End main()
-  }
-  
-  main()
+}
+
+main()
```

Comment:
If this is running in nodejs I think the program would end with an unresolved promise

Reply:
Thanks, I think I fixed this in c829b7d